src/server/createRoutes.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+const globby = require('globby');
+const debug = require('./debug');
+const express = require('express');
+const mainRouter = express.Router();
+
+async function createRoutes (routesDir) {
+    try {
+        const files = await globby('**/*.js', { cwd: path.resolve(routesDir) });
+
+        files
+            .filter(filename => !filename.endsWith('-test.js'))
+            .map(filename => ({
+                routePrefix: path.dirname(filename),
+                Router:      require(path.join('../../', routesDir, filename))
+            }))
+            .filter(({ Router }) => Object.getPrototypeOf(Router) === express.Router)
+            .map(({
+                routePrefix,
```

Comment:
Please, do not write arrow function arguments on multiple lines. It looks complex.

Reply:
If an arrow can't fit into the line, it's better to split its body. Or event refactor it into a named function or method.